x-pack/legacy/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/beats/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,307 @@
+      values: {
+        beatType: beatType || UNDETECTED_BEAT_TYPE
+      }
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <EuiLink
+            href={`${ELASTIC_WEBSITE_URL}guide/en/beats/metricbeat/${DOC_LINK_VERSION}/metricbeat-installation.html`}
+            target="_blank"
+          >
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.beatsInstructions.installMetricbeatLinkText"
+              defaultMessage="Follow the instructions here"
+            />
+          </EuiLink>
+        </p>
+      </EuiText>
+    )
+  };
+
+  const httpEndpointUrl = `${ELASTIC_WEBSITE_URL}guide/en/beats/${beatType || DEFAULT_BEAT_FOR_URLS}`
+    + `/${DOC_LINK_VERSION}/http-endpoint.html`;
+
+  const enableMetricbeatModuleStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.beatsInstructions.enableMetricbeatModuleTitle', {
+      defaultMessage: 'Enable and configure the Beat x-pack module in Metricbeat'
+    }),
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+          metricbeat modules enable beat-xpack
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.beatsInstructions.enableMetricbeatModuleDescription"
+              defaultMessage="By default the module will collect {beatType} monitoring metrics from http://localhost:5066. If the {beatType} instance being monitored has a different address, you must specify it via the {hosts} setting in the {file} file."
+              values={{
+                hosts: (
+                  <Monospace>hosts</Monospace>
+                ),
+                file: (
+                  <Monospace>modules.d/beat-xpack.yml</Monospace>
+                ),
+                beatType: beatType || UNDETECTED_BEAT_TYPE
+              }}
+            />
+          </p>
+        </EuiText>
+        <EuiSpacer size="m"/>
+        <EuiCallOut
+          color="warning"
+          iconType="help"
+          title={(
+            <EuiText>
+              <p>
+                <FormattedMessage
+                  id="xpack.monitoring.metricbeatMigration.beatsInstructions.enableMetricbeatModuleHttpEnabledDirections"
+                  defaultMessage="In order for Metricbeat to collect metrics from the running {beatType}, you need to {link}."
+                  values={{
+                    link: (
+                      <EuiLink
+                        href={httpEndpointUrl}
+                        target="_blank"
+                      >
+                        <FormattedMessage
+                          id="xpack.monitoring.metricbeatMigration.beatsInstructions.enableMetricbeatModuleHttpEnabledDirectionsLinkText"
+                          defaultMessage="enable an HTTP endpoint for the {beatType} instance being monitored"
```

Comment:
Doesn't this need to start with a capital E? `Enable an...` and also end with a period?

Reply:
It's contained within a sentence already. Look at the `<FormattedMessage>` for `xpack.monitoring.metricbeatMigration.beatsInstructions.enableMetricbeatModuleHttpEnabledDirections` right above this

Reply:
Ah, I see it now, thanks for clarifying!